index.html

```diff
@@ -1,14 +1,14 @@
 <script type="module">
     import {Crypto} from "./dist/api.js"
-    import {LocalStoragePersister} from "./dist/persister.js"
 
     function sleep(ms) {
         return new Promise(resolve => setTimeout(resolve, ms));
     }
 
-    // *****WARNING*****: DO NOT USE LocalStoragePersister for any purpose other than testing locally
-    let persister = new LocalStoragePersister();
-    const c = new Crypto("<>", persister);
+    // By default, JS SDK uses in-memory persister.
```

Comment:
I have to pass in a null persister to get this default behavior? Is this more secure? Why? :)